libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/RefreshCallback.java

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute;
+
+public interface RefreshCallback {
+  void onRefresh(DirectionsRoute directionsRoute);
+
+  void onError(String error);
```

Comment:
Would it make sense to return a "refresh error" object (instead of a `String`)? This is what we implemented for other APIs e.g. https://github.com/mapbox/mapbox-navigation-android/blob/master/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/OnOfflineRouteFoundCallback.java